Tests/SkillFunctionalTests/Utils/README.md

```diff
@@ -0,0 +1,22 @@
+# Skills Functional Tests Utilities Scripts
+
+## ConfigureAppSettings
+
+This script configures the _appsettings.Development.json_ file from the SkillsFunctionalTest project to be able to run the tests locally connecting them with bots deployed in Azure. Its behavior consists on gathering the DirectLine Secrets from the deployed Azure Bot Resources and configure this file with them. For its purpose it uses the '**ResourceGroup**' and '**ResourceSuffix**' provided by the user when running it.
+
+>**ResourceGroup**: the specific Resource Group where the Azure Resoruce Bots are deployed in. The default value is 'BFFN'. _Eg: bffnbots_
+>
+>**ResourceSuffix**: the suffix the resources name are built with. It will be a combination of the suffix itself and the build id. _Eg: gitali-218_

```

Comment:
```suggestion
>**ResourceSuffix**: the suffix the resources name is built with. It will be a combination of the suffix itself and the build id where the resource was created. _Eg: gitali-218_

```